gb-backend/src/main/groovy/nl/thehyve/gb/backend/client/AbstractRestClient.groovy

```diff
@@ -39,11 +36,13 @@ abstract class AbstractRestClient {
     protected <T> T postOnBehalfOf(String impersonatedUserName, URI uri, Map<String, Object> body, Class<T> type) {
         log.info "User impersonation! User ${authContext.user.username} " +
                 "sending request on behalf of user $impersonatedUserName, requestURL: $uri"
+        RestTemplate restTemplate = new RestTemplate()
```

Comment:
This one does not have a possibility to disable the trust manager as `getRestTemplate()` does? Maybe just move `getRestTemplate()` in here?

Reply:
Why is this disabling required for transmart? In case gb-backend and transmart are deployed on separate servers?